qutebrowser/components/caretcommands.py

```diff
@@ -209,3 +209,11 @@ def follow_selected(tab_obj: apitypes.Tab, *, tab: bool = False) -> None:
         tab_obj.caret.follow_selected(tab=tab)
     except apitypes.WebTabError as e:
         raise cmdutils.CommandError(str(e))
+
+
+@cmdutils.register()
```

Comment:
This should use `modes=[cmdutils.KeyMode.caret])` because it only really makes sense to run this command in caret mode - I'll add it while merging though.